src/css/style.css

```diff
@@ -67,7 +67,7 @@ body {
   border: 2px solid #c4c4c4;
   box-sizing: border-box;
   border-radius: 10px;
-  padding-left: 35px;
```

Comment:
I think this padding is the reason why the text is not centered, please try removing it or adding equal padding to both sides and the text will be centered.

Reply:
there already use text-align:center; bt the result is same.i used equal padding to both sides.